### ISSUE TITLE
Unknown props warning

### ISSUE THREAD

**Issue by landvibe** [coderhaoxin/react-flatpickr#4]
when I set minDate or maxDate properties, it shows below warning message.
```
bundle.js:10659 Warning: Unknown props `minDate`, `maxDate` on <input> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
```

**Comment by coderhaoxin** [comment]
I should remove these props before passing to `input` tag.

**Comment by landvibe** [comment]
there is another warning
```
warning.js:36Warning: Failed form propType: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`. Check the render method of `DateTimePicker`
```

This PR resolves above warnings
#5

**Comment by coderhaoxin** [comment]
fixed by 73dd29a31d9bad0848105e3864803a87f2da9260

PR https://github.com/coderhaoxin/react-flatpickr/pull/5
